test/verify/check-connection

```diff
@@ -1045,6 +1045,26 @@ until pgrep -f '^/usr/bin/cockpit-bridge.*--privileged'; do sleep 1; done
         b.wait_visible("#login")
         b.assert_pixels("body", "login-screen")
 
+    def test_auth_unix_path(self):
+        '''test UnixPath for auth method in cockpit.conf'''
+        m = self.machine
+
+        m.execute(['systemctl', 'start', 'cockpit-session.socket'])
```

Comment:
This [fails](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16819-20230104-122301-a9528565-fedora-coreos/log.html#331) on coreos:
```
systemd[1]: Starting cockpit-session.socket - Initiator socket for Cockpit sessions...
systemd[1014]: cockpit-session.socket: Failed to resolve group cockpit-wsinstance: No such process
systemd[1]: cockpit-session.socket: Control process exited, code=exited, status=216/GROUP
```
That doesn't exist because the cockpit-ws package isn't installed on coreos. We need to skip that test here.

Reply:
Added a skip.